src/ontology/uberon-edit.obo

```diff
@@ -9394,7 +9394,6 @@ xref: Wikipedia:Neck
 is_a: UBERON:0000475 ! organism subdivision
 relationship: anteriorly_connected_to UBERON:0000033 ! head
 relationship: dubious_for_taxon NCBITaxon:32443 {source="ISBN:9780674021839"} ! Teleostei
-relationship: only_in_taxon NCBITaxon:8287 ! Sarcopterygii
 relationship: part_of UBERON:0000153 ! anterior region of body
 relationship: part_of UBERON:0007811 ! craniocervical region
```

Comment:
Given [this comment](https://github.com/obophenotype/uberon/issues/2417#issuecomment-1169217139), should `relationship: part_of UBERON:0007811 ! craniocervical region` be removed?